syntax = "proto3";

service TestService {
    rpc SendFile (FileRequest) returns (FileResponse);
}

message FileRequest {
    bytes file_content = 1;
}
message FileResponse {
    string base64_data = 1;
}